Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM ubuntu
+
+WORKDIR /performance-dashboards
+ARG DEBIAN_FRONTEND=noninteractive
+
+# Install necessary libraries for subsequent commands
+RUN apt-get update && apt-get install -y podman dumb-init python3.6 python3-distutils python3-pip python3-apt
+
+COPY . .
+RUN chmod -R 775 /performance-dashboards
+
+# Install dependencies
+RUN python3 -m pip install --upgrade pip
+RUN pip install -r requirements.txt
+
+# Cleanup the installation remainings
+RUN apt-get clean autoclean && \
+    apt-get autoremove --yes && \
+    rm -rf /var/lib/{apt,dpkg,cache,log}/
```

Comment:
These lines are not necessary and don't make any effect, as the layer from apt cache is still part of the container image.

Reply:
Removed them as they do not have any significant affect in this case.